crates/core/src/dht/dht_store.rs

```diff
@@ -49,6 +50,9 @@ pub struct DhtStore {
     /// All the entry aspects that the network has told us to hold
     holding_map: AspectMap,
 
+    /// Hold aspect requests from the network
+    queued_hold_aspect_requests: HashMap<ProcessUniqueId, Result<(), HolochainError>>,
```

Comment:
Confused about the name. Should this be something like `holding_attempt_results`?